examples/federated-learning/tff/nvflare/README.md

```diff
@@ -0,0 +1,283 @@
+# Nvflare with TensorFlow
+
+This example uses Nvflare to train an image classifier using federated averaging and TensorFlow as the deep learning framework.
+
+[NVIDIA FLARE](https://nvflare.readthedocs.io/en/main/index.html) (**NV**IDIA **F**ederated **L**earning **A**pplication **R**untime **E**nvironment)
+is a domain-agnostic, open-source, extensible SDK that allows researchers and data scientists to adapt existing ML/DL workflows to a federated paradigm.
+It enables platform developers to build a secure, privacy-preserving offering for a distributed multi-party collaboration.
+
+[Here](https://nvflare.readthedocs.io/en/main/flare_overview.html#high-level-system-architecture) is a high level system architecture of Nvflare.
+
+> **_NOTE:_** This example uses the [MNIST](http://yann.lecun.com/exdb/mnist/) handwritten digits dataset and will load its data within the trainer code.
+
+This example builds on top of the infrastructure that the
+[blueprint provides](../../../../README.md), and follows the best practices the
+blueprint establishes.
+
+## High-level diagram
+This demo relies on one server and two clients that are connected to the server through a secure gRPC link.
+
+Here is a high-level diagram:
+
+![alt_text](../../../../assets/nvflare.svg "Infrastructure overview")
```

Comment:
The architecture diagram is not clear because we don't specify what the various boxes are. Refactor it to show that they are Pods and containers inside these Pods.